src/Stoa.ts

```diff
@@ -232,7 +231,7 @@ class Stoa extends WebService
                 // Nothing to show
                 if (!rows.length)
                 {
-                    res.status(204).send();
+                    res.status(400).send("The validator data not found.");
```

Comment:
How about adding `address`?

Reply:
I don't know what it means. Give me a suggestion.

Reply:
`The validator data not found. ${address}`

Reply:
I think this would be better.
`The validator data not found. (address: ${address}, height: ${height})`

Reply:
The requester will of course know the value of the parameter.
I don't think it's a bad idea to mark it.